packages/cli/src/commands/webhook.ts

```diff
@@ -99,56 +71,23 @@ export class Webhook extends Command {
 			this.error('Webhook processes can only run with execution mode as queue.');
```

Comment:
There's something a bit odd - this message is not being displayed although it's exiting because of this.